README.md

```diff
@@ -6,22 +6,91 @@
+  validity*](https://zebra.zfnd.org/dev/rfcs/0002-parallel-verification.html#verification-stages) by making invalid states unrepresentable.  For instance, the
+  `Transaction` enum has variants for each transaction version, and it's
+  impossible to construct a transaction with, e.g., spend or output
+  descriptions but no binding signature, or, e.g., a version 2 (Sprout)
+  transaction with Sapling proofs.  Currently, `zebra-chain` is oriented
+  towards verifying transactions, but will be extended to support creating them
+  in the future.
 
-[Join us on Discord](https://discord.gg/na6QZNd).
+- [`zebra-network`](https://doc.zebra.zfnd.org/zebra_network/index.html),
+  providing an asynchronous, multithreaded implementation of the Zcash network
+  protocol inherited from Bitcoin.  In contrast to `zcashd`, each peer
+  connection has a separate state machine, and the crate translates the
+  external network protocol into a stateless, request/response-oriented
+  protocol for internal use.  The crate provides two interfaces: an
+  auto-managed connection pool that load-balances requests over available
+  peers, and a `connect_isolated` method that produces a peer connection
+  completely isolated from all other node state.  This can be used, for
+  instance, to safely relay data over Tor, without revealing distinguishing
+  information.
+
+- [`zebra-script`](https://doc.zebra.zfnd.org/zebra_script/index.html) provides
+  script validation.  Currently, this is implemented by linking to the C++
+  script verification code from `zcashd`, but in the future we may implement a
+  pure-Rust Script implementation.
+
+- [`zebra-consensus`](https://doc.zebra.zfnd.org/zebra_consensus/index.html)
+  performs *semantic validation* of blocks and transactions: all consensus
+  rules that can be checked independently of the chain state, such as
+  verification of signatures, proofs, and scripts.  Internally, the library
+  uses [`tower-batch`](https://doc.zebra.zfnd.org/tower_batch/index.html) to
+  perform automatic, transparent batch processing of contemporaneous
+  verification requests.
+
+- [`zebra-state`](https://doc.zebra.zfnd.org/zebra_state/index.html) is
+  responsible for storing, updating, and querying the chain state.  The state
+  service is responsible for *contextual verification*: all consensus rules
+  that check whether a new block is a valid extension of an existing chain,
+  such as updating the nullifier set or checking that transaction inputs remain
+  unspent.
+  
+- [`zebrad`](https://doc.zebra.zfnd.org/zebrad/index.html) contains the full
+  node, which connects these components together and implements logic to handle
+  inbound requests from peers and the chain sync process.
+
+- `zebra-rpc` and `zebra-client` will eventually contain the RPC and wallet
+  functionality, but as mentioned above, our goal is to implement replication
+  of chain state first before asking users to entrust it with their funds.
```

Comment:
Do we want to say "entrust us" or "entrust Zebra"?
```suggestion
  of chain state first before asking users to entrust Zebra with their funds.
```

Reply:
Entrust Zebra SGTM